Source/UI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -253,3 +266,26 @@ extension BillingFormViewController: CountrySelectionViewControllerDelegate {
         delegate?.update(country: country)
     }
 }
+
+extension BillingFormViewController: UIScrollViewDelegate {
+  func scrollViewDidScroll(_ scrollView: UIScrollView) {
+    var contentOffsetY = scrollView.contentOffset.y
+
+    if #available(iOS 11.0, *) {
```

Comment:
Pull other changes from recent merges and this `if #available` should not be needed.